.github/workflows/gh-ci-tests.yaml

```
@@ -0,0 +1,55 @@
+name: mda_gh_ci_tests
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+
```

```suggestion

concurrency:
  # Probably overly cautious group naming.
  # Commits to develop/master will cancel each other, but PRs will only cancel
  # commits within the same PR
  group: "${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}"
  cancel-in-progress: true

```
Add concurrency check